ws-nextjs-app/pages/[service]/[[...]].page.tsx

```diff
@@ -0,0 +1,29 @@
+import { GetServerSideProps } from 'next';
+import { Services, Variants } from '../../../src/app/models/types/global';
+
+// This route does nothing other than return a 404 status code for other routes not yet supported in the Next.JS app
+export default function CatchAll() {
+  return null;
+}
+
+type PageDataParams = {
+  service: Services;
+  variant: Variants;
+};
+
+export const getServerSideProps: GetServerSideProps = async context => {
+  const { service, variant } = context.query as PageDataParams;
+
+  const { headers: reqHeaders } = context.req;
+
+  return {
+    props: {
+      bbcOrigin: reqHeaders['bbc-origin'] || null,
+      isNextJs: true,
+      service,
+      status: 404,
+      timeOnServer: Date.now(), // TODO: check if needed?
```

Comment:
Not sure if this TODO is outside the scope of this PR.

timeOnServer seems to only be used in the CanonicalRadioSchedule to calculate the programs state. I believe it sole purpose is for users to not inject their machine clock and change the schedule's status client side. (No security threat if they are allowed to, I think it is just a product decision)

Might come in handy for consistent now time for the future, but unless pages that display radioschedule are diplsayed with next.js we might not currently need them?